docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,7 +1,352 @@
 ---
 title: Setting Up a Flow Access Node
 sidebar_title: Access Node Setup
-description: How to run a Permissionless Access node
 ---
 
-🏗️ Coming soon on 8th Feb, 2023
\ No newline at end of file
+This guide is for running a permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](node-bootstrap.mdx).
+
+Permissionless Access nodes allow any operator to run a Flow Access node.
+Unlike the other staked nodes, a permissionless access node does not have to be approved by the service account before it can join the network, hence the term "permissionless". The goal is to make all node types permissionless and this is the first step towards achieving that goal.
+
+## Who should run a Permissionless Access node?
+dApp developers can choose to run their own private permissionless access node and move away from using the community access nodes. This will also allow them to not be subjected to the API rate limits of the public access nodes.
+
+Node operators can also run their own permissionless access node and provide access to that node as a service.
+
+Chain analytics, audit and exploration applications can run such an access node and do not have to rely on third parties for the state of the network.
+
+
+## Timing
+
+New nodes are able to join the network each time a new epoch begins.
+An epoch is a period of time (approximately one week) when the node operators in the network are constant.
+At epoch boundaries, newly staked node operators are able to join the network and existing node operators which have unstaked may exit the network.
+You can read more about epochs [here](https://developers.flow.com/nodes/staking/schedule).
+
+
+In order to join the network at epoch N+1, the access node **must** be registered with at least 100 FLOW staked prior to the end of epoch N's Staking Auction Phase.
+
+Currently on mainnet, the staking auction starts every Wednesday at around 17:30 UTC and ends on the next Wednesday at 8:30 UTC.
+Hence, the node should be staked by _Wednesday, 08:30 UTC_ to be able to join the network in the next epoch.
```

Comment:
```suggestion
Currently on mainnet, the staking auction starts every Wednesday at around 20:00 UTC and ends on the next Wednesday at around 12:00 UTC. 
Since this deadline may shift slightly from epoch to epoch, we recommend the node be staked by _Wednesday, 8:00 UTC_ to be able to join the network in the next epoch.
```

- 12:00 PT is 20:00UTC when I run the conversion. 
- The target time for staking auction end is 4AM PT or 12:00UTC. I'm OK to either pad this deadline like you have (say 8:00UTC) or to give the real deadline and add a note that the deadlines shift epoch to epoch (added this as suggestion)

Reply:
@jordanschalm am confused. On grafana, I see different time in UTC - 
![image](https://user-images.githubusercontent.com/1117327/217352546-3a519062-c93c-4a6d-8f7a-b0f530a25f41.png)

Reply:
That graph is showing the actual phase change time, for one epoch (the last one). Last week the transition was earlier than the target -- over the course of the last week we've adjusted, and we're tracking for about a 11:30UTC staking auction end (target is 12:00UTC).